snekbox/nsjail.py

```diff
@@ -171,12 +171,21 @@ def _consume_stdout(nsjail: subprocess.Popen) -> str:
 
         return "".join(output)
 
-    def python3(self, code: str, *args) -> CompletedProcess:
+    def python3(
+        self,
+        code: str,
+        *,
+        nsjail_args: Iterable[str] = (),
+        py_args: Iterable[str] = ("-c",)
+    ) -> CompletedProcess:
         """
         Execute Python 3 code in an isolated environment and return the completed process.
 
         Additional arguments passed will be used to override the values in the NsJail config.
```

Comment:
Revise this to explicitly mention `nsjail_args` rather than "additional arguments".